packages/react-native-editor/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ For each user feature we should also add a importance categorization label  to i
 -->
 
 ## Unreleased
+-   [**] [Embed block] Implement WP embed preview component [#34004]
```

Comment:
I added the entry for the WP embed preview component implementation because it was missing.